libr/bin/bin.c

```diff
@@ -31,6 +31,13 @@ R_LIB_VERSION (r_bin);
 
 #define ARCHS_KEY "archs"
 
+#if !defined(R_BIN_STATIC_PLUGINS)
```

Comment:
Yes, I had a look at that and didn't see a simple solution to fix it in configure-plugins, so I worked around the problem by adding the ifdef instead.